poetry/utils/env.py

```diff
@@ -1002,7 +1003,20 @@ def build_venv(
 
         args.append(str(path))
 
-        return virtualenv.cli_run(args)
+        cli_result = virtualenv.cli_run(args)
+
+        print("path", str(path))
```

Comment:
Did you mean to leave these prints in? I'm guessing they are leftovers from debugging?